community/index/src/main/java/org/neo4j/index/internal/gbptree/SeekCursor.java

```diff
@@ -513,7 +513,7 @@ private boolean internalNext() throws IOException
                 if ( verifyExpectedFirstAfterGoToNext )
                 {
                     pos = seekForward ? 0 : keyCount - 1;
-                    bTreeNode.keyAt( cursor,firstKeyInNode, pos, INTERNAL );
```

Comment:
Hah, was thinking about the method name `internalNext()`, that clashes with LEAF,INTERNAL, doesn't it? Stumbled upon it when reading this change and thinking why the heck you'd want to read a LEAF key when you're doing a next for an INTERNAL node... before it struck me :)

Reply:
Haha, yeah that's an unfortunate clash